engines/bladerunner/fog.cpp

```diff
@@ -115,7 +115,8 @@ void Fog::setupFrame(int frame) {
 void FogSphere::read(Common::ReadStream *stream, int frameCount) {
 	_frameCount = frameCount;
 	int size = readCommon(stream);
-	_radius = stream->readFloatLE();
+	float radius = stream->readFloatLE();
+	_radius_e2 = radius * radius;
```

Comment:
This is a minor issue, but for readability, could you name these squared variables with an "_sq" suffix or a "_sq_" prefix, rather than the "_e2" suffix?